src/emulator/functionsEmulator.ts

```diff
@@ -659,7 +659,7 @@ export class FunctionsEmulator implements EmulatorInstance {
         return true;
       })
       .catch((err) => {
-        this.logger.log("WARN", "Error adding trigger: " + err);
+        this.logger.log("WARN", "Error adding rtdb trigger: " + err);
```

Comment:
s/rtdb/realtime database

These aren't employees and they don't know the slang.